test/JDBC/expected/GRANT_SCHEMA.out

```diff
@@ -6628,3 +6628,39 @@ go
 
 drop database babel_5172_db
 go
+
+-- check the catalog entry when the table name and proc name are same.
+create proc GRANT_SCHEMA_t1 as select 1;
+go
+grant execute on GRANT_SCHEMA_t1 to guest;
+go
+create table GRANT_SCHEMA_t1(a int);
+go
+grant select on GRANT_SCHEMA_t1 to guest;
+go
+
+-- psql
+select schema_name, object_name, permission, grantee, grantor from sys.babelfish_schema_permissions
+where object_name = 'GRANT_SCHEMA_t1' collate sys.database_default;
+go
+~~START~~
+"sys"."varchar"#!#"sys"."varchar"#!#int4#!#"sys"."varchar"#!#"sys"."varchar"
+dbo#!#grant_schema_t1#!#128#!#master_guest#!#<NULL>
+dbo#!#grant_schema_t1#!#2#!#master_guest#!#<NULL>
```

Comment:
One doubt. If I have two databases db1 and db2 containing table name t1, they will have the exact same entries in catalog right (apart from grantee name)? IMO we should persist physical schema name in catalog rather than the logical schema name.

Reply:
we store database id in the catalog as well to distinguish.

Reply:
Oh right missed that